### DISCUSSION TITLE
Audio beep

### DISCUSSION THREAD

**Discussion by Galaxy-Man** [google/oboe#2056] [General]
Before I delve into the OBOE libary I dont want to waste resources looking into something that cannot be done or already done.

Does anyone know if it is possible to play a beep during a phone call in the native dialer using audio upstream and downstream on an Android Mobile device such as the Samsung Galaxy?

Any help would be appreciated. This may have to be completed in Kotlin or Native C++.

Thank you everyone

**Comment by philburk** [comment]
If you develop your own VoIP app then you can add a beep to the local or remote audio.

But you cannot add a beep to the remote audio when using the **native dialer** using Oboe. I don't know of any API for that.

**Reply by Galaxy-Man** [reply]
Hi Phil
VoIP is not an option for me. This is something I'll have to continue discussions with the manufacturer and Google. 
Thanks for the reply.
